Jenkinsfile

```diff
@@ -453,45 +340,25 @@ pipeline {
                         }
                     }
                     steps {
-                        githubNotify credentialsId: 'daos-jenkins-commit-status',
-                                      description: env.STAGE_NAME,
-                                      context: "build" + "/" + env.STAGE_NAME,
-                                      status: "PENDING"
-                        checkoutScm withSubmodules: true
-                        catchError(stageResult: 'UNSTABLE',
-                                   buildResult: 'SUCCESS') {
-                            sh label: env.STAGE_NAME,
-                               script: 'ci/rpm_build.sh'
-                        }
+                        buildRpm unstable: true
```

Comment:
We shouldn't allow this stage to fail and test stages to be run as the RPMs that fail to build here will fail to be installed in the test stage(s).  That is was being done before was an error.  This would be a good opportunity to fix that.